frontend/src/metabase/dashboard/components/Dashboard/DashboardEmptyState/DashboardEmptyState.tsx

```diff
@@ -1,79 +1,92 @@
-import cx from "classnames";
-import { t } from "ttag";
+import type { ReactNode } from "react";
+import { c, jt, t } from "ttag";
 
-import EmptyState from "metabase/components/EmptyState";
-import Button from "metabase/core/components/Button";
-import Link from "metabase/core/components/Link";
-import CS from "metabase/css/core/index.css";
-import { isEmbeddingSdk } from "metabase/env";
-import * as Urls from "metabase/lib/urls";
-import type { Dashboard } from "metabase-types/api";
+import EmptyDashboardBot from "assets/img/dashboard-empty.svg";
+import { Button, Icon, type IconName, Stack, Text, Title } from "metabase/ui";
 
-import { Container, QuestionCircleStyled } from "./DashboardEmptyState.styled";
+interface DashboardEmptyStateProps {
+  addQuestion?: () => void;
+  isDashboardEmpty: boolean;
+  isEditing?: boolean;
+  isNightMode: boolean;
+}
+
+const getDefaultTitle = (isDashboardEmpty: boolean) =>
+  isDashboardEmpty ? t`This dashboard is empty` : t`There's nothing here, yet`;
 
-function QuestionIllustration() {
-  return <QuestionCircleStyled>?</QuestionCircleStyled>;
+function InlineIcon({ name }: { name: IconName }) {
+  return <Icon name={name} style={{ verticalAlign: " middle" }} />;
```

Comment:
Really didn't feel like adding a new CSS module just because of this. Hope that an isolated `style` prop is ok here?

Reply:
Ouch 😊 

Thank you